### PR TITLE
Updating the help content for v6.

### PR DESCRIPTION
**Pull request by adityapatwardhan**
Version(s) of document impacted
------------------------------
- [X] Impacts 6 document
- [ ] Impacts 5.1 document
- [ ] Impacts 5.0 document
- [ ] Impacts 4.0 document
- [ ] Impacts 3.0 document

Reason(s) for not updating all version of documents
--------------------------------------------------
- [X] The documented feature was introduced in version (list version here) of PowerShell
- [ ] This issue only shows up in version (list version(s) here) of the document
- [ ] This PR partially fixes the issue, and issue #<insert here> tracks the remaining work

### CODE REVIEW

**Comment by markekraus** [comment]
@adityapatwardhan The main title and subtitle of the article still reference `PowerShell.exe.`

File: reference/6/Microsoft.PowerShell.Core/About/about_pwsh.md [hunk 1]
```diff
@@ -110,44 +106,39 @@ Minimized, Maximized and Hidden.
...
-Troubleshooting note: In Windows PowerShell 2.0, starting some programs
-from the Windows PowerShell console fails with a LastExitCode of 0xc0000142.
-
 # EXAMPLES
 
-PowerShell -Version
+pwsh -Version
 
-PowerShell -Command {Get-EventLog -LogName security}
+pwsh -Command {Get-EventLog -LogName security}
```
Comment (markekraus) [inline reference/6/Microsoft.PowerShell.Core/About/about_pwsh.md]: Perhaps we could use this opportunity to change this to a cross-platform example? `Get-Process -Name pwsh` maybe?

**Comment by saldana** [comment]
### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/6/Microsoft.PowerShell.Core/About/about_pwsh.md](https://github.com/adityapatwardhan/PowerShell-Docs/blob/FixPwshAboutHelp/reference/6/Microsoft.PowerShell.Core/About/about_pwsh.md) | :warning:Warning | | [Details](#user-content-reference%2F6%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_pwsh.md)
[reference/6/Microsoft.PowerShell.Core/About/about_PowerShell_Ise_exe.md](https://github.com/adityapatwardhan/PowerShell-Docs/blob/FixPwshAboutHelp/reference/6/Microsoft.PowerShell.Core/About/about_PowerShell_Ise_exe.md) | :warning:Warning | | [Details](#user-content-reference%2F6%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_PowerShell_Ise_exe.md)

<a name="reference%2F6%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_pwsh.md"></a>
### [reference/6/Microsoft.PowerShell.Core/About/about_pwsh.md](https://github.com/adityapatwardhan/PowerShell-Docs/blob/FixPwshAboutHelp/reference/6/Microsoft.PowerShell.Core/About/about_pwsh.md)
 - **[Warning]** Ignore metadata: locale. They are generated by open publish.

<a name="reference%2F6%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_PowerShell_Ise_exe.md"></a>
### [reference/6/Microsoft.PowerShell.Core/About/about_PowerShell_Ise_exe.md](https://github.com/adityapatwardhan/PowerShell-Docs/blob/FixPwshAboutHelp/reference/6/Microsoft.PowerShell.Core/About/about_PowerShell_Ise_exe.md)
 - **[Warning]** Ignore metadata: locale. They are generated by open publish.
 - **Line 105**: **[Warning]** Invalid file link:(~/6/Microsoft.PowerShell.Core/About/about_PowerShell_exe.md).

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C11%5C14%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201711142147599738-1853%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=1T4gw3KasS45wN7lBF5DPijC9%2FP55lEHS0m3rm41Ypw%3D&st=2017-11-14T22%3A14%3A02Z&se=2017-12-15T22%3A19%3A02Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by saldana** [comment]
### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/6/Microsoft.PowerShell.Core/About/About.md](https://github.com/adityapatwardhan/PowerShell-Docs/blob/FixPwshAboutHelp/reference/6/Microsoft.PowerShell.Core/About/About.md) | :warning:Warning | | [Details](#user-content-reference%2F6%2FMicrosoft.PowerShell.Core%2FAbout%2FAbout.md)
[reference/6/Microsoft.PowerShell.Core/About/about_PowerShell_Ise_exe.md](https://github.com/adityapatwardhan/PowerShell-Docs/blob/FixPwshAboutHelp/reference/6/Microsoft.PowerShell.Core/About/about_PowerShell_Ise_exe.md) | :warning:Warning | | [Details](#user-content-reference%2F6%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_PowerShell_Ise_exe.md)
[reference/6/Microsoft.PowerShell.Core/About/about_pwsh.md](https://github.com/adityapatwardhan/PowerShell-Docs/blob/FixPwshAboutHelp/reference/6/Microsoft.PowerShell.Core/About/about_pwsh.md) | :warning:Warning | | [Details](#user-content-reference%2F6%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_pwsh.md)

<a name="reference%2F6%2FMicrosoft.PowerShell.Core%2FAbout%2FAbout.md"></a>
### [reference/6/Microsoft.PowerShell.Core/About/About.md](https://github.com/adityapatwardhan/PowerShell-Docs/blob/FixPwshAboutHelp/reference/6/Microsoft.PowerShell.Core/About/About.md)
 - **[Warning]** Can&#39;t find service name for 6/Microsoft.PowerShell.Core/About/About.md
 - **[Warning]** Can&#39;t find service name for 6/Microsoft.PowerShell.Core/About/About.md
 - **[Warning]** Can&#39;t find service name for 6/Microsoft.PowerShell.Core/About/About.md
 - **[Warning]** Can&#39;t find service name for 6/Microsoft.PowerShell.Core/About/About.md
 - **[Warning]** Can&#39;t find service name for 6/Microsoft.PowerShell.Core/About/About.md
 - **[Warning]** Can&#39;t find service name for 6/Microsoft.PowerShell.Core/About/About.md
 - **[Warning]** Can&#39;t find service name for 6/Microsoft.PowerShell.Core/About/About.md
 - **[Warning]** Can&#39;t find service name for 6/Microsoft.PowerShell.Core/About/About.md
 - **[Warning]** Can&#39;t find service name for 6/Microsoft.PowerShell.Core/About/About.md
 - **[Warning]** Can&#39;t find service name for 6/Microsoft.PowerShell.Core/About/About.md
 - **[Warning]** Can&#39;t find service name for 6/Microsoft.PowerShell.Core/About/About.md
 - **[Warning]** Can&#39;t find service name for 6/Microsoft.PowerShell.Core/About/About.md
 - **[Warning]** Can&#39;t find service name for 6/Microsoft.PowerShell.Core/About/About.md
 - **[Warning]** Can&#39;t find service name for 6/Microsoft.PowerShell.Core/About/About.md
 - **[Warning]** Can&#39;t find service name for 6/Microsoft.PowerShell.Core/About/About.md
 - **[Warning]** Can&#39;t find service name for 6/Microsoft.PowerShell.Core/About/About.md
 - **[Warning]** Can&#39;t find service name for 6/Microsoft.PowerShell.Core/About/About.md
 - **[Warning]** Can&#39;t find service name for 6/Microsoft.PowerShell.Core/About/About.md
 - **[Warning]** Can&#39;t find service name for 6/Microsoft.PowerShell.Core/About/About.md
 - **[Warning]** Can&#39;t find service name for 6/Microsoft.PowerShell.Core/About/About.md
 - **[Warning]** Can&#39;t find service name for 6/Microsoft.PowerShell.Core/About/About.md
 - **[Warning]** Can&#39;t find service name for 6/Microsoft.PowerShell.Core/About/About.md
 - **[Warning]** Can&#39;t find service name for 6/Microsoft.PowerShell.Core/About/About.md
 - **[Warning]** Can&#39;t find service name for 6/Microsoft.PowerShell.Core/About/About.md
 - **[Warning]** Can&#39;t find service name for 6/Microsoft.PowerShell.Core/About/About.md
 - **[Warning]** Can&#39;t find service name for 6/Microsoft.PowerShell.Core/About/About.md
 - **[Warning]** Can&#39;t find service name for 6/Microsoft.PowerShell.Core/About/About.md
 - **[Warning]** Can&#39;t find service name for 6/Microsoft.PowerShell.Core/About/About.md
 - **[Warning]** Can&#39;t find service name for 6/Microsoft.PowerShell.Core/About/About.md
 - **[Warning]** Can&#39;t find service name for 6/Microsoft.PowerShell.Core/About/About.md
 - **[Warning]** Can&#39;t find service name for 6/Microsoft.PowerShell.Core/About/About.md
 - **[Warning]** Can&#39;t find service name for 6/Microsoft.PowerShell.Core/About/About.md
 - **[Warning]** Can&#39;t find service name for 6/Microsoft.PowerShell.Core/About/About.md
 - **[Warning]** Can&#39;t find service name for 6/Microsoft.PowerShell.Core/About/About.md
 - **[Warning]** Can&#39;t find service name for 6/Microsoft.PowerShell.Core/About/About.md
 - **[Warning]** Can&#39;t find service name for 6/Microsoft.PowerShell.Core/About/About.md
 - **[Warning]** Can&#39;t find service name for 6/Microsoft.PowerShell.Core/About/About.md
 - **[Warning]** Can&#39;t find service name for 6/Microsoft.PowerShell.Core/About/About.md
 - **[Warning]** Can&#39;t find service name for 6/Microsoft.PowerShell.Core/About/About.md
 - **[Warning]** Can&#39;t find service name for 6/Microsoft.PowerShell.Core/About/About.md
 - **[Warning]** Can&#39;t find service name for 6/Microsoft.PowerShell.Core/About/About.md
 - **[Warning]** Can&#39;t find service name for 6/Microsoft.PowerShell.Core/About/About.md
 - **[Warning]** Can&#39;t find service name for 6/Microsoft.PowerShell.Core/About/About.md
 - **[Warning]** Can&#39;t find service name for 6/Microsoft.PowerShell.Core/About/About.md
 - **[Warning]** Can&#39;t find service name for 6/Microsoft.PowerShell.Core/About/About.md
 - **[Warning]** Can&#39;t find service name for 6/Microsoft.PowerShell.Core/About/About.md
 - **[Warning]** Can&#39;t find service name for 6/Microsoft.PowerShell.Core/About/About.md
 - **[Warning]** Can&#39;t find service name for 6/Microsoft.PowerShell.Core/About/About.md
 - **[Warning]** Can&#39;t find service name for 6/Microsoft.PowerShell.Core/About/About.md
 - **[Warning]** Can&#39;t find service name for 6/Microsoft.PowerShell.Core/About/About.md
 - **[Warning]** Can&#39;t find service name for 6/Microsoft.PowerShell.Core/About/About.md
 - **[Warning]** Can&#39;t find service name for 6/Microsoft.PowerShell.Core/About/About.md
 - **[Warning]** Can&#39;t find service name for 6/Microsoft.PowerShell.Core/About/About.md
 - **[Warning]** Can&#39;t find service name for 6/Microsoft.PowerShell.Core/About/About.md
 - **[Warning]** Can&#39;t find service name for 6/Microsoft.PowerShell.Core/About/About.md
 - **[Warning]** Can&#39;t find service name for 6/Microsoft.PowerShell.Core/About/About.md
 - **[Warning]** Can&#39;t find service name for 6/Microsoft.PowerShell.Core/About/About.md
 - **[Warning]** Can&#39;t find service name for 6/Microsoft.PowerShell.Core/About/About.md
 - **[Warning]** Can&#39;t find service name for 6/Microsoft.PowerShell.Core/About/About.md
 - **[Warning]** Can&#39;t find service name for 6/Microsoft.PowerShell.Core/About/About.md
 - **[Warning]** Can&#39;t find service name for 6/Microsoft.PowerShell.Core/About/About.md
 - **[Warning]** Can&#39;t find service name for 6/Microsoft.PowerShell.Core/About/About.md
 - **[Warning]** Can&#39;t find service name for 6/Microsoft.PowerShell.Core/About/About.md
 - **[Warning]** Can&#39;t find service name for 6/Microsoft.PowerShell.Core/About/About.md
 - **[Warning]** Can&#39;t find service name for 6/Microsoft.PowerShell.Core/About/About.md
 - **[Warning]** Can&#39;t find service name for 6/Microsoft.PowerShell.Core/About/About.md
 - **[Warning]** Can&#39;t find service name for 6/Microsoft.PowerShell.Core/About/About.md
 - **[Warning]** Can&#39;t find service name for 6/Microsoft.PowerShell.Core/About/About.md
 - **[Warning]** Can&#39;t find service name for 6/Microsoft.PowerShell.Core/About/About.md
 - **[Warning]** Can&#39;t find service name for 6/Microsoft.PowerShell.Core/About/About.md
 - **[Warning]** Can&#39;t find service name for 6/Microsoft.PowerShell.Core/About/About.md
 - **[Warning]** Can&#39;t find service name for 6/Microsoft.PowerShell.Core/About/About.md
 - **[Warning]** Can&#39;t find service name for 6/Microsoft.PowerShell.Core/About/About.md
 - **[Warning]** Can&#39;t find service name for 6/Microsoft.PowerShell.Core/About/About.md
 - **[Warning]** Can&#39;t find service name for 6/Microsoft.PowerShell.Core/About/About.md
 - **[Warning]** Can&#39;t find service name for 6/Microsoft.PowerShell.Core/About/About.md
 - **[Warning]** Can&#39;t find service name for 6/Microsoft.PowerShell.Core/About/About.md
 - **[Warning]** Can&#39;t find service name for 6/Microsoft.PowerShell.Core/About/About.md
 - **[Warning]** Can&#39;t find service name for 6/Microsoft.PowerShell.Core/About/About.md
 - **[Warning]** Can&#39;t find service name for 6/Microsoft.PowerShell.Core/About/About.md
 - **[Warning]** Can&#39;t find service name for 6/Microsoft.PowerShell.Core/About/About.md
 - **[Warning]** Can&#39;t find service name for 6/Microsoft.PowerShell.Core/About/About.md
 - **[Warning]** Can&#39;t find service name for 6/Microsoft.PowerShell.Core/About/About.md
 - **[Warning]** Can&#39;t find service name for 6/Microsoft.PowerShell.Core/About/About.md
 - **[Warning]** Can&#39;t find service name for 6/Microsoft.PowerShell.Core/About/About.md
 - **[Warning]** Can&#39;t find service name for 6/Microsoft.PowerShell.Core/About/About.md
 - **[Warning]** Can&#39;t find service name for 6/Microsoft.PowerShell.Core/About/About.md
 - **[Warning]** Can&#39;t find service name for 6/Microsoft.PowerShell.Core/About/About.md
 - **[Warning]** Can&#39;t find service name for 6/Microsoft.PowerShell.Core/About/About.md
 - **[Warning]** Can&#39;t find service name for 6/Microsoft.PowerShell.Core/About/About.md
 - **[Warning]** Can&#39;t find service name for 6/Microsoft.PowerShell.Core/About/About.md
 - **[Warning]** Can&#39;t find service name for 6/Microsoft.PowerShell.Core/About/About.md
 - **[Warning]** Can&#39;t find service name for 6/Microsoft.PowerShell.Core/About/About.md
 - **[Warning]** Can&#39;t find service name for 6/Microsoft.PowerShell.Core/About/About.md
 - **[Warning]** Can&#39;t find service name for 6/Microsoft.PowerShell.Core/About/About.md
 - **[Warning]** Can&#39;t find service name for 6/Microsoft.PowerShell.Core/About/About.md
 - **[Warning]** Can&#39;t find service name for 6/Microsoft.PowerShell.Core/About/About.md
 - **[Warning]** Can&#39;t find service name for 6/Microsoft.PowerShell.Core/About/About.md
 - **[Warning]** Can&#39;t find service name for 6/Microsoft.PowerShell.Core/About/About.md
 - **[Warning]** Can&#39;t find service name for 6/Microsoft.PowerShell.Core/About/About.md
 - **[Warning]** Can&#39;t find service name for 6/Microsoft.PowerShell.Core/About/About.md
 - **[Warning]** Can&#39;t find service name for 6/Microsoft.PowerShell.Core/About/About.md
 - **[Warning]** Can&#39;t find service name for 6/Microsoft.PowerShell.Core/About/About.md
 - **[Warning]** Can&#39;t find service name for 6/Microsoft.PowerShell.Core/About/About.md

<a name="reference%2F6%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_PowerShell_Ise_exe.md"></a>
### [reference/6/Microsoft.PowerShell.Core/About/about_PowerShell_Ise_exe.md](https://github.com/adityapatwardhan/PowerShell-Docs/blob/FixPwshAboutHelp/reference/6/Microsoft.PowerShell.Core/About/about_PowerShell_Ise_exe.md)
 - **[Warning]** Ignore metadata: locale. They are generated by open publish.

<a name="reference%2F6%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_pwsh.md"></a>
### [reference/6/Microsoft.PowerShell.Core/About/about_pwsh.md](https://github.com/adityapatwardhan/PowerShell-Docs/blob/FixPwshAboutHelp/reference/6/Microsoft.PowerShell.Core/About/about_pwsh.md)
 - **[Warning]** Ignore metadata: locale. They are generated by open publish.

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C11%5C14%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201711142248404343-1853%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=7jUnYbV7M6Uiay0hLicWBKGxdvOEz39vJzCy2S1LXQk%3D&st=2017-11-14T23%3A22%3A31Z&se=2017-12-15T23%3A27%3A31Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by adityapatwardhan** [comment]
Added @sdwheeler to merge when ready.

**Comment by adityapatwardhan** [comment]
@sdwheeler any ideas when this merges to live?

**Comment by sdwheeler** [comment]
@adityapatwardhan The merge to live is manual. I will do a PR later today.

**Pull request merged**